tests/check_check_sub.c

```diff
@@ -267,7 +295,9 @@ START_TEST(test_ck_assert_int_lt)
 {
   int x = 2;
   int y = 3;
+
```

Comment:
What adding the spaces necessary for C89 support? Or is this a whitespace only change?

Reply:
It's not required by the standard. The reason was:

I made all the changes in tests/check_check_sub.c manually, and they were quite a lot.
So I finally came up with a "let's have all functions look like this":

```
START_TEST(test_something) 
{
  type1 var1;
  type2 var2;

  record_test_name(tcase_name());

  test code
  more test code
}
END_TEST
```

Just to get this structure at first sight. This helped me to keep an overview what is already handled and what is not. Maybe it's helpful for maintenance as well.

But it's your project, so choose whatever structure you like.

Do you actually have something like a `clang-format` file for automatic code formatting?

Reply:
I'm fine with the whitespace changes, just curious. I do not have a clang-format file for formatting.